prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -169,7 +173,7 @@ class Hl7Serializer(val metadata: Metadata) {
             }
             // add the rows
             if (parsedValue.isNullOrEmpty()) {
-                warnings.add("Blank for $terserSpec")
+                warnings.add("Blank for $terserSpec - $elementName")
             }
```

Comment:
Added the element name to the warning so it's easier to see what fields are empty.